### ISSUE TITLE
evaluation metric

### ISSUE THREAD

**Issue by xiaobh1519** [NVlabs/OmniDrive#9]
Hi，I replicated your code and trained the model for six cycles using the same data. However, when I evaluated the model's performance, the L2 and collision metrics were between those of Omnidrive and Omnidrive++ in Table 5, but the Boundary metric was significantly worse, far exceeding any cases in Table 5. Could you please explain why this is happening? Have you encountered similar results before?
![image](https://github.com/user-attachments/assets/d79e0896-cfdf-4e3b-93d1-9409853ee8bd)
![image](https://github.com/user-attachments/assets/5f29d470-3252-4efe-a481-94ee63293565)

**Comment by exiawsh** [comment]
I will retrain the model and check. I use an existing checkpoint trained before and the results are normal.

**Comment by exiawsh** [comment]
And Please send me your config file.

**Comment by xiaobh1519** [comment]
> And Please send me your training log.

Sent. Please check!

**Comment by exiawsh** [comment]
I noticed that you have increased the batch size from 2 to 4. My suggestion is to maintain the current batch size 2. Or you may need to increase your learning rate.

**Comment by mo704507064** [comment]
The training code is released.

> Hi，I replicated your code and trained the model for six epochs using the same data. However, when I evaluated the model's performance, the L2 and collision metrics were between those of Omnidrive and Omnidrive++ in Table 5, but the Boundary metric was significantly worse, far exceeding any cases in Table 5. Could you please explain why this is happening? Have you encountered similar results before? ![image](https://private-user-images.githubusercontent.com/79265019/351805386-d79e0896-cfdf-4e3b-93d1-9409853ee8bd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5OTI2MTMsIm5iZiI6MTcyMTk5MjMxMywicGF0aCI6Ii83OTI2NTAxOS8zNTE4MDUzODYtZDc5ZTA4OTYtY2ZkZi00ZTNiLTkzZDEtOTQwOTg1M2VlOGJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI2VDExMTE1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIzNWY0MTJiNzNmMDg4NDYzMzZmOTUwYjIyZGZhYTZhNDBhNmExZTcyYTQ1ZjMyMWMxOGU2ZTczYTQ5ZWViMzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.czplkgIdY3zQKmacdu63LptK2nY6i-gGRGhZiv8BwZ0) ![image](https://private-user-images.githubusercontent.com/79265019/351805625-5f29d470-3252-4efe-a481-94ee63293565.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5OTI2MTMsIm5iZiI6MTcyMTk5MjMxMywicGF0aCI6Ii83OTI2NTAxOS8zNTE4MDU2MjUtNWYyOWQ0NzAtMzI1Mi00ZWZlLWE0ODEtOTRlZTYzMjkzNTY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI2VDExMTE1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViYjMyZmZmMDU4NDZkOWZjYTBlNDY5OGI1NmM3YTQ1ZTE5OWUzNzk2OGVlMDkxY2Y1NjUxZjQxOTlhNWJlNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.epCIPHaxFy5reqNHXLvRFiTOxqNtfMYhziyA6p_88zI)

The training code is released? can you tell me in which file?

**Comment by missTL** [comment]
> Hi，I replicated your code and trained the model for six epochs using the same data. However, when I evaluated the model's performance, the L2 and collision metrics were between those of Omnidrive and Omnidrive++ in Table 5, but the Boundary metric was significantly worse, far exceeding any cases in Table 5. Could you please explain why this is happening? Have you encountered similar results before? ![image](https://github.com/user-attachments/assets/d79e0896-cfdf-4e3b-93d1-9409853ee8bd) ![image](https://github.com/user-attachments/assets/5f29d470-3252-4efe-a481-94ee63293565)

Have you got a reasonable Boundary metric?
